src/components/ec-metroline-card/ec-metroline-card.spec.js

```diff
@@ -0,0 +1,43 @@
+import { mount } from '@vue/test-utils';
+import EcMetrolineCard from './ec-metroline-card.vue';
+
+describe('EcMetrolineCard', () => {
+  function mountMetrolineCard(opts, mountOpts) {
+    return mount(EcMetrolineCard, {
+      propsData: {
+        ...opts,
+      },
+      ...mountOpts,
+    });
+  }
+
+  it('should render as expected', async () => {
+    const wrapper = mountMetrolineCard();
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should render as expected when is collapsed', async () => {
+    const wrapper = mountMetrolineCard({ isCollapsed: true });
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should render as expected when has narrow padding', async () => {
+    const wrapper = mountMetrolineCard({ hasNarrowPadding: true });
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should render as expected when is collapsed and has narrow padding', async () => {
```

Comment:
as commented above this could be inside one of those describes

Reply:
Updated